mobx_codegen/test/data/valid_import_prefixed_output.dart

```diff
@@ -1,10 +1,17 @@
 mixin _$User<T extends io.Process> on UserBase<T>, Store {
-  Computed<io.File> _$biographyNotesComputed;
+  Computed<io.File>? _$biographyNotesComputed;
```

Comment:
In general we should have a plain, nullable and then inner nullable types as in:

- `io.File`
- `io.File?`
- `List<io.File>?`
- `List<io.File?>`
- `List<io.File?>?`

Reply:
done